_data/navigation.yml

```diff
@@ -2,12 +2,14 @@
 main:
   - title: "Call for Papers"
     url: /calls/papers
+  - title: "Call for Demos"
```

Comment:
Since we have multiple categories of calls now, we should revert back to the structure which can be seen in https:[//2021.eacl.org,](url) where the main navigation bar links to "Calls", which contains subpages for Papers, Demos, and other things in the future.

Reply:
Fixed! I've also modified the hierarchy of sections in the call for main papers to match the one in EACL 2021. While testing, I found that the TOC was getting clipped because the call for papers had too many `h2`s.